### ISSUE TITLE
Add little dots for the location of cities

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1571]
In a certain zoom level the names of big cities are shown but there are no dots for the location of the cities. In countries you don't know so well this is irritating because the names could also belong to the regions. When showing dots for the cities it is important to make them small enough not to be mixed up with the other dots. 

It would be nice to have the names of the regions displayed as well. This would be especially usefull when you search for the regions to give to existing entries.

**Comment by trugwaldsaenger** [comment]
Agreed that this would be helpfull and that it is not super important. I guess we can keep it with P3. Eventually @formimkontext and @j0hj0h can keep this in the back of the head, when looking at map-libraries the next time...
